### PR TITLE
recipe creation now through a modal that redirects to edition mode

### PR DESCRIPTION
**Pull request by jessicaarrais**

### CODE REVIEW

File: src/components/Cookbook.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export default function Cookbook(props: CookbookProps) {
   return (
     <>
       <S.Header>
-        {props.createRecipeButton}
+        {props.newRecipeModal}
```
Comment (davicatunda) [inline src/components/Cookbook.tsx:36]: this is still a button, why did you change the name

File: src/components/CreateRecipeButton.tsx [hunk 2]
```diff
@@ -1,23 +1,53 @@
...
+}
+
+function CreateRecipeButton(props: Props) {
+  const history = useHistory();
+
+  const [createRecipe, { error }] = useMutation<CreateRecipeReponse>(CREATE_RECIPE, {
+    onCompleted(data) {
+      const urlTitle = urlParser(data.createRecipe.recipe.title);
+      history.push(
+        `/${'editing'}/cookbook/${
```
Comment (davicatunda) [inline src/components/CreateRecipeButton.tsx:45]: `/${'editing'}/cookbook/${
        `/editing/cookbook/${

**Review by davicatunda — APPROVED** [review]

File: src/components/Cookbook.tsx (hunk 1 above)
Comment (jessicaarrais) [reply src/components/Cookbook.tsx:36]: because this is the button that opens the modal and CreateRecipeButton is the one wich saves the recipe.
